src/test-utils/dom/internal/chart-popover.ts

```diff
@@ -4,6 +4,25 @@ import { ComponentWrapper, ElementWrapper } from '@cloudscape-design/test-utils-
 import ButtonWrapper from '../button';
 import styles from '../../../internal/components/chart-popover/styles.selectors.js';
 import popoverStyles from '../../../popover/styles.selectors.js';
+import chartSeriesDetails from '../../../internal/components/chart-series-details/styles.selectors.js';
+
+export class ChartPopoverSeriesItemWrapper extends ComponentWrapper {
+  findKey(): ElementWrapper | null {
+    return this.findByClassName(chartSeriesDetails.key);
+  }
+
+  findValue(): ElementWrapper | null {
+    return this.findByClassName(chartSeriesDetails.value);
+  }
+}
+
+export class ChartPopoverSeriesWrapper extends ChartPopoverSeriesItemWrapper {
+  findSubItems(): Array<ChartPopoverSeriesItemWrapper> {
```

Comment:
1. Do we really need to differentiate between items and sub-items in test utils?
2. Would it make sense to add utils like getItemByIndex(index) or getItemByTitle(title)?

Reply:
The testing utils were part of the API review so I would prefer to not deviate unless having a strong reason. To the specific questions:

1. I see a clear hierarchy between top-level items, which represent actual series displayed in the graph, and their sub-items, which are arbitrarily defined by the customers, so I think an API that reflects that makes sense. And consider the case where sub-items are expandable, it's not clear how a flat-list approach should work (should all items always be returned? If not, should the results update and the indices change when expanding/collapsing?) and it would be more confusing IMO.
2. Regarding `getItemByIndex`, you can already access the items by index in unit tests or use `.get(0)` in integration tests, so would it bring much? Regarding `getItemByTitle`, maybe it wouldn't hurt, but I'm not sure iI would add it before having specific data about its need.